### ISSUE TITLE
Uncaught TypeError: Cannot read property '0' of undefined

### ISSUE THREAD

**Issue by ab674** [jesseweed/seti-ui#424]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.17.0 x64
**Electron**: 1.3.15
**OS**: Mac OS X 10.12.4
**Thrown From**: [seti-ui](https://github.com/jesseweed/seti-ui) package 1.8.0

### Stack Trace

Uncaught TypeError: Cannot read property '0' of undefined

```
At /Users/abhibassi/.atom/packages/seti-ui/lib/headers.coffee:8

TypeError: Cannot read property '0' of undefined
    at updateTreeViewHeaderPosition (/packages/seti-ui/lib/headers.coffee:8:23)
    at /packages/seti-ui/lib/headers.coffee:34:19
```

### Commands

```
     -9:17.6.0 core:backspace (input.hidden-input)
     -7:47.2.0 fuzzy-finder:toggle-file-finder (input.hidden-input)
  2x -7:41.9.0 core:backspace (input.hidden-input)
```

### Non-Core Packages

```
seti-syntax 1.1.3 
seti-ui 1.8.0 
```

**Comment by eatobin** [comment]
Same issue on Linux...

**Comment by MyloTuT** [comment]
I removed a project folder and it triggered this error.

**Comment by Xuanwo** [comment]
The same here on Ubuntu 16.04.2 with atom 1.17

```
/home/xuanwo/.atom/packages/seti-ui/lib/headers.coffee:8
Hide Stack Trace
TypeError: Cannot read property '0' of undefined
    at updateTreeViewHeaderPosition (/home/xuanwo/.atom/packages/seti-ui/lib/headers.coffee:8:23)
    at /home/xuanwo/.atom/packages/seti-ui/lib/headers.coffee:29:5
    at Function.module.exports.Emitter.simpleDispatch (/opt/atom/resources/app/node_modules/event-kit/lib/emitter.js:25:1)
    at Emitter.module.exports.Emitter.emit (/opt/atom/resources/app/node_modules/event-kit/lib/emitter.js:141:1)
    at Project.module.exports.Project.removePath (/opt/atom/resources/app/src/project.js:322:1)
    at TreeView.module.exports.TreeView.removeProjectFolder (/opt/atom/resources/app/node_modules/tree-view/lib/tree-view.js:1252:1)
    at HTMLDivElement.atom.commands.add.tree-view:remove-project-folder (/opt/atom/resources/app/node_modules/tree-view/lib/tree-view.js:385:1)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/opt/atom/resources/app/src/command-registry.js:265:1)
    at CommandRegistry.handleCommandEvent (/opt/atom/resources/app/src/command-registry.js:3:1)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/opt/atom/resources/app/src/command-registry.js:166:1)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchContextMenuCommand (/opt/atom/resources/app/src/atom-environment.js:1292:1)
    at EventEmitter.outerCallback (/opt/atom/resources/app/src/application-delegate.js:347:1)
    at emitThree (events.js:116:13)
    at EventEmitter.emit (events.js:194:7)
```

**Comment by howessteve** [comment]
Changing Theme Color in seti-ui settings causes this for me on Atom v1.17 on Mac

**Comment by covard** [comment]
Same issue with update `1.17.1` on macOS `10.12.4`

**Comment by mgween** [comment]
[Enter steps to reproduce:]

1. Right-click a project folder.
2. Remove Project Folder

**Atom**: 1.17.2 x64
**Electron**: 1.3.15
**OS**: Microsoft Windows 10 Home
**Thrown From**: [seti-ui](https://github.com/jesseweed/seti-ui) package 1.8.0


### Stack Trace

Uncaught TypeError: Cannot read property '0' of undefined

```
At C:\Users\Matt\.atom\packages\seti-ui\lib\headers.coffee:8

TypeError: Cannot read property '0' of undefined
    at updateTreeViewHeaderPosition (/packages/seti-ui/lib/headers.coffee:8:23)
    at /packages/seti-ui/lib/headers.coffee:29:5
    at Function.module.exports.Emitter.simpleDispatch (~/AppData/Local/atom/app-1.17.2/resources/app/node_modules/event-kit/lib/emitter.js:25:1)
    at Emitter.module.exports.Emitter.emit (~/AppData/Local/atom/app-1.17.2/resources/app/node_modules/event-kit/lib/emitter.js:141:1)
    at Project.module.exports.Project.removePath (~/AppData/Local/atom/app-1.17.2/resources/app/src/project.js:322:1)
    at TreeView.module.exports.TreeView.removeProjectFolder (~/AppData/Local/atom/app-1.17.2/resources/app/node_modules/tree-view/lib/tree-view.js:1252:1)
    at HTMLDivElement.atom.commands.add.tree-view:remove-project-folder (~/AppData/Local/atom/app-1.17.2/resources/app/node_modules/tree-view/lib/tree-view.js:385:1)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (~/AppData/Local/atom/app-1.17.2/resources/app/src/command-registry.js:265:1)
    at CommandRegistry.handleCommandEvent (~/AppData/Local/atom/app-1.17.2/resources/app/src/command-registry.js:3:1)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (~/AppData/Local/atom/app-1.17.2/resources/app/src/command-registry.js:166:1)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchContextMenuCommand (~/AppData/Local/atom/app-1.17.2/resources/app/src/atom-environment.js:1306:1)
    at EventEmitter.outerCallback (~/AppData/Local/atom/app-1.17.2/resources/app/src/application-delegate.js:347:1)
    at emitThree (events.js:116:13)
    at EventEmitter.emit (events.js:194:7)
```

### Commands

```
 14x -6:34.3.0 core:move-right (input.hidden-input)
     -6:32.7.0 core:backspace (input.hidden-input)
  2x -6:31.6.0 intentions:highlight (input.hidden-input)
     -6:31.2.0 core:save (input.hidden-input)
 11x -6:26.3.0 intentions:highlight (input.hidden-input)
     -6:25.5.0 core:save (input.hidden-input)
     -4:06.6.0 editor:newline (input.hidden-input)
     -3:58.7.0 core:move-right (input.hidden-input)
     -3:45.2.0 core:select-up (input.hidden-input)
     -3:44.3.0 core:backspace (input.hidden-input)
     -2:08.4.0 git-plus:diff-all (input.hidden-input)
     -1:25.7.0 pane:close (div.line)
     -1:20.3.0 git-plus:add-all-and-commit (input.hidden-input)
     -1:00.7.0 intentions:highlight (input.hidden-input)
     -1:00.6.0 core:save (input.hidden-input)
     -0:45.3.0 tree-view:remove-project-folder (div.header.list-item.project-root-header)
```

### Non-Core Packages

```
atom-beautify 0.29.24 
atom-bracket-highlight 0.3.2 
atom-material-syntax 1.0.6 
atom-typescript 11.0.3 
busy-signal 1.4.1 
file-icons 2.1.6 
git-plus 7.9.2 
highlight-selected 0.13.1 
intentions 1.1.2 
language-typescript-grammars-only 1.6.0 
linter 2.1.4 
linter-ui-default 1.6.0 
pigments 0.39.1 
seti-ui 1.8.0 
```

**Comment by Xuanwo** [comment]
After upgrade atom to 1.17.2, this problem solved. 
So it's may be a atom issue which not related seti-ui.
And this issue could be closed.

@jesseweed

**Comment by covard** [comment]
I can confirm @Xuanwo is correct upgrading to `1.17.2` fixes this issue.

**Comment by BryanHunt** [comment]
I'm still seeing this issue in `1.17.2` on macOS.

**Comment by eatobin** [comment]
I also still have this issue in `1.17.2` on SuSE Linux after removing a project folder...

**Comment by jeffwurz** [comment]
have this issue on windows 7, I dragged a project from one window to another and when i changed focus to the destination window, it threw the error.

**Comment by benlipp** [comment]
Getting this issue on Ubuntu 16.04 with Atom `1.17.2` when changing seti-ui settings in Atom.

**Comment by mgween** [comment]
Still having this issue.

[Enter steps to reproduce:]

1. Remove Project Folder

**Atom**: 1.17.2 x64
**Electron**: 1.3.15
**OS**: Microsoft Windows 10 Home
**Thrown From**: [seti-ui](https://github.com/jesseweed/seti-ui) package 1.8.0


### Stack Trace

Uncaught TypeError: Cannot read property '0' of undefined

```
At C:\Users\Matt\.atom\packages\seti-ui\lib\headers.coffee:8

TypeError: Cannot read property '0' of undefined
    at updateTreeViewHeaderPosition (/packages/seti-ui/lib/headers.coffee:8:23)
    at /packages/seti-ui/lib/headers.coffee:29:5
    at Function.module.exports.Emitter.simpleDispatch (~/AppData/Local/atom/app-1.17.2/resources/app/node_modules/event-kit/lib/emitter.js:25:1)
    at Emitter.module.exports.Emitter.emit (~/AppData/Local/atom/app-1.17.2/resources/app/node_modules/event-kit/lib/emitter.js:141:1)
    at Project.module.exports.Project.removePath (~/AppData/Local/atom/app-1.17.2/resources/app/src/project.js:322:1)
    at TreeView.module.exports.TreeView.removeProjectFolder (~/AppData/Local/atom/app-1.17.2/resources/app/node_modules/tree-view/lib/tree-view.js:1252:1)
    at HTMLDivElement.atom.commands.add.tree-view:remove-project-folder (~/AppData/Local/atom/app-1.17.2/resources/app/node_modules/tree-view/lib/tree-view.js:385:1)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (~/AppData/Local/atom/app-1.17.2/resources/app/src/command-registry.js:265:1)
    at CommandRegistry.handleCommandEvent (~/AppData/Local/atom/app-1.17.2/resources/app/src/command-registry.js:3:1)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (~/AppData/Local/atom/app-1.17.2/resources/app/src/command-registry.js:166:1)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchContextMenuCommand (~/AppData/Local/atom/app-1.17.2/resources/app/src/atom-environment.js:1306:1)
    at EventEmitter.outerCallback (~/AppData/Local/atom/app-1.17.2/resources/app/src/application-delegate.js:347:1)
    at emitThree (events.js:116:13)
    at EventEmitter.emit (events.js:194:7)
```

### Commands

```
     -2:07.7.0 editor:newline (input.hidden-input)
     -2:07.6.0 intentions:highlight (input.hidden-input)
     -2:07.4.0 core:paste (input.hidden-input)
  3x -2:05.8.0 core:move-up (input.hidden-input)
     -2:05.0 core:move-right (input.hidden-input)
     -2:04.8.0 core:move-down (input.hidden-input)
 15x -2:04.6.0 core:move-right (input.hidden-input)
 10x -2:03.0 core:backspace (input.hidden-input)
     -1:59 core:move-down (input.hidden-input)
  3x -1:58.7.0 core:move-right (input.hidden-input)
 14x -1:57.1.0 core:backspace (input.hidden-input)
     -1:54.7.0 core:move-right (input.hidden-input)
  6x -1:53.2.0 core:move-left (input.hidden-input)
  3x -1:50.6.0 core:move-right (input.hidden-input)
     -1:30.1.0 editor:newline (input.hidden-input)
     -1:05.8.0 tree-view:remove-project-folder (span.name.icon.icon-repo)
```

### Non-Core Packages

```
atom-beautify 0.29.26 
atom-bracket-highlight 0.3.2 
atom-material-syntax 1.0.6 
atom-typescript 11.0.3 
busy-signal 1.4.1 
file-icons 2.1.6 
git-plus 7.9.2 
highlight-selected 0.13.1 
intentions 1.1.2 
language-typescript-grammars-only 1.6.0 
linter 2.1.4 
linter-ui-default 1.6.0 
pigments 0.39.1 
seti-ui 1.8.0 
```

**Comment by OmgImAlexis** [comment]
[Enter steps to reproduce:]

1. Same as @mgween

**Atom**: 1.17.2 x64
**Electron**: 1.3.15
**OS**: Mac OS X 10.12.3
**Thrown From**: [seti-ui](https://github.com/jesseweed/seti-ui) package 1.8.0


### Stack Trace

Uncaught TypeError: Cannot read property '0' of undefined

```
At /Users/xo/.atom/packages/seti-ui/lib/headers.coffee:8

TypeError: Cannot read property '0' of undefined
    at updateTreeViewHeaderPosition (/packages/seti-ui/lib/headers.coffee:8:23)
    at /packages/seti-ui/lib/headers.coffee:29:5
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:1)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:1)
    at Project.module.exports.Project.removePath (/Applications/Atom.app/Contents/Resources/app/src/project.js:322:1)
    at TreeView.module.exports.TreeView.removeProjectFolder (/Applications/Atom.app/Contents/Resources/app/node_modules/tree-view/lib/tree-view.js:1252:1)
    at HTMLDivElement.atom.commands.add.tree-view:remove-project-folder (/Applications/Atom.app/Contents/Resources/app/node_modules/tree-view/lib/tree-view.js:385:1)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:265:1)
    at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:3:1)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:166:1)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchContextMenuCommand (/Applications/Atom.app/Contents/Resources/app/src/atom-environment.js:1306:1)
    at EventEmitter.outerCallback (/Applications/Atom.app/Contents/Resources/app/src/application-delegate.js:347:1)
    at emitThree (events.js:116:13)
    at EventEmitter.emit (events.js:194:7)
```

### Commands

```
     -4:47 snippets:expand (input.hidden-input)
     -4:46.9.0 editor:indent (input.hidden-input)
     -4:45.3.0 snippets:next-tab-stop (input.hidden-input)
     -4:45.3.0 snippets:expand (input.hidden-input)
     -4:45.3.0 editor:indent (input.hidden-input)
     -4:43.6.0 snippets:next-tab-stop (input.hidden-input)
     -4:43.6.0 snippets:expand (input.hidden-input)
     -4:43.6.0 editor:indent (input.hidden-input)
     -4:43 core:save (input.hidden-input)
     -4:35 core:copy (input.hidden-input)
     -4:29.8.0 core:paste (input.hidden-input)
     -4:29.5.0 core:save (input.hidden-input)
     -3:48.9.0 core:select-all (input.hidden-input)
     -3:48.6.0 core:copy (input.hidden-input)
     -0:14.8.0 tree-view:remove (span.name.icon.icon-repo)
     -0:09.8.0 tree-view:remove-project-folder (span.name.icon.icon-repo)
```

### Non-Core Packages

```
autohide-tree-view 0.25.1 
busy-signal 1.4.1 
duplicate-removal 0.1.3 
editorconfig 2.2.2 
file-icons 2.1.6 
git-status 0.3.4 
intentions 1.1.2 
language-docker 1.1.8 
language-mako 0.6.0 
language-nginx 0.8.0 
language-pug 0.0.21 
language-vue 0.22.0 
linter 2.1.4 
linter-pycodestyle 2.0.2 
linter-stylelint 3.10.0 
linter-swagger 0.4.1 
linter-ui-default 1.6.0 
linter-xo 0.23.0 
multi-cursor 2.1.5 
seti-syntax 1.1.3 
seti-ui 1.8.0 
sort-lines 0.14.0 
tree-view-autoresize 2.0.0 
tree-view-git-status 1.4.0 
```

**Comment by OmgImAlexis** [comment]
[Enter steps to reproduce:]

Same thing again

**Atom**: 1.17.2 x64
**Electron**: 1.3.15
**OS**: Mac OS X 10.12.3
**Thrown From**: [seti-ui](https://github.com/jesseweed/seti-ui) package 1.8.0


### Stack Trace

Uncaught TypeError: Cannot read property '0' of undefined

```
At /Users/xo/.atom/packages/seti-ui/lib/headers.coffee:8

TypeError: Cannot read property '0' of undefined
    at updateTreeViewHeaderPosition (/packages/seti-ui/lib/headers.coffee:8:23)
    at /packages/seti-ui/lib/headers.coffee:29:5
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:1)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:1)
    at Project.module.exports.Project.removePath (/Applications/Atom.app/Contents/Resources/app/src/project.js:322:1)
    at TreeView.module.exports.TreeView.removeProjectFolder (/Applications/Atom.app/Contents/Resources/app/node_modules/tree-view/lib/tree-view.js:1252:1)
    at HTMLDivElement.atom.commands.add.tree-view:remove-project-folder (/Applications/Atom.app/Contents/Resources/app/node_modules/tree-view/lib/tree-view.js:385:1)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:265:1)
    at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:3:1)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:166:1)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchContextMenuCommand (/Applications/Atom.app/Contents/Resources/app/src/atom-environment.js:1306:1)
    at EventEmitter.outerCallback (/Applications/Atom.app/Contents/Resources/app/src/application-delegate.js:347:1)
    at emitThree (events.js:116:13)
    at EventEmitter.emit (events.js:194:7)
```

### Commands

```
     -1:50.1.0 tree-view:show (atom-workspace.workspace.scrollbars-visible-when-scrolling.seti-compact.seti-icons.seti-roboto.seti-animate)
     -0:30 editor:newline (input.hidden-input)
     -0:29.9.0 core:paste (input.hidden-input)
  2x -0:28.7.0 editor:select-to-first-character-of-line (input.hidden-input)
  2x -0:28.3.0 core:backspace (input.hidden-input)
  2x -0:27.9.0 core:save (input.hidden-input)
     -0:04.1.0 tree-view:remove-project-folder (div.header.list-item.project-root-header)
```

### Non-Core Packages

```
autohide-tree-view 0.25.1 
busy-signal 1.4.1 
duplicate-removal 0.1.3 
editorconfig 2.2.2 
file-icons 2.1.6 
git-status 0.3.4 
intentions 1.1.2 
language-docker 1.1.8 
language-mako 0.6.0 
language-nginx 0.8.0 
language-pug 0.0.21 
language-vue 0.22.0 
linter 2.1.4 
linter-pycodestyle 2.0.2 
linter-stylelint 3.10.0 
linter-swagger 0.4.1 
linter-ui-default 1.6.0 
linter-xo 0.23.0 
multi-cursor 2.1.5 
seti-syntax 1.1.3 
seti-ui 1.8.0 
sort-lines 0.14.0 
tree-view-autoresize 2.0.0 
tree-view-git-status 1.4.0 
```

**Comment by jesseweed** [comment]
I'm seeing this intermittently again as well. I haven't pinpointed the cause yet, but I'll looking into it.

**Comment by andreiwow2** [comment]
Opened the editor, it told me that the editor failed and I pressed on keep waiting, once the editor finally started I got this error.
`[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.17.2 x64
**Electron**: 1.3.15
**OS**: Microsoft Windows 10 Pro
**Thrown From**: [seti-ui](https://github.com/jesseweed/seti-ui) package 1.8.0


### Stack Trace

Uncaught TypeError: Cannot read property '0' of undefined

```
At C:\Users\andrei\.atom\packages\seti-ui\lib\headers.coffee:8

TypeError: Cannot read property '0' of undefined
    at updateTreeViewHeaderPosition (/packages/seti-ui/lib/headers.coffee:8:23)
    at /packages/seti-ui/lib/headers.coffee:29:5
    at Function.module.exports.Emitter.simpleDispatch (~/AppData/Local/atom/app-1.17.2/resources/app/node_modules/event-kit/lib/emitter.js:25:1)
    at Emitter.module.exports.Emitter.emit (~/AppData/Local/atom/app-1.17.2/resources/app/node_modules/event-kit/lib/emitter.js:141:1)
    at Project.module.exports.Project.removePath (~/AppData/Local/atom/app-1.17.2/resources/app/src/project.js:322:1)
    at TreeView.module.exports.TreeView.removeProjectFolder (~/AppData/Local/atom/app-1.17.2/resources/app/node_modules/tree-view/lib/tree-view.js:1252:1)
    at HTMLDivElement.atom.commands.add.tree-view:remove-project-folder (~/AppData/Local/atom/app-1.17.2/resources/app/node_modules/tree-view/lib/tree-view.js:385:1)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (~/AppData/Local/atom/app-1.17.2/resources/app/src/command-registry.js:265:1)
    at CommandRegistry.handleCommandEvent (~/AppData/Local/atom/app-1.17.2/resources/app/src/command-registry.js:3:1)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (~/AppData/Local/atom/app-1.17.2/resources/app/src/command-registry.js:166:1)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchContextMenuCommand (~/AppData/Local/atom/app-1.17.2/resources/app/src/atom-environment.js:1306:1)
    at EventEmitter.outerCallback (~/AppData/Local/atom/app-1.17.2/resources/app/src/application-delegate.js:347:1)
    at emitThree (events.js:116:13)
    at EventEmitter.emit (events.js:194:7)
```

### Commands

```
 15x -1:19.9.0 intentions:highlight (input.hidden-input)
     -1:19 tree-view:toggle (input.hidden-input)
     -1:09.4.0 tree-view:remove-project-folder (span.name.icon.icon-file-directory)
     -1:04.4.0 application:add-project-folder (ol.full-menu.list-tree.has-collapsable-children.focusable-panel)
```

### Non-Core Packages

```
aligner-php 1.1.1 
atom-autocomplete-php 0.22.2 
atom-beautify 0.29.26 
autoclose-html 0.23.0 
busy-signal 1.4.3 
color-picker 2.2.5 
double-tag 0.8.0 
expose 0.14.0 
github-issues 0.8.3 
gitlab 0.1.3-7 
goto-definition 1.3.3 
highlight-selected 0.13.1 
intentions 1.1.2 
linter 2.1.4 
linter-csslint 1.3.4 
linter-jscs 4.1.2 
linter-php 1.3.2 
linter-ui-default 1.6.0 
minimap 4.28.2 
minimap-cursorline 0.2.0 
minimap-linter 2.0.0 
pigments 0.39.1 
Remote-FTP 0.10.7 
save-session 0.15.7 
seti-ui 1.8.0 
todo-show 1.11.0 
tool-bar 1.1.0 
tool-bar-almighty 0.8.2 
wakatime 7.0.3 
```

`

**Comment by jesseweed** [comment]
I think have a fix for this. Will and try and push an update soon.

**Comment by OmgImAlexis** [comment]
[Enter steps to reproduce:]

@jesseweed still getting this

**Atom**: 1.17.2 x64
**Electron**: 1.3.15
**OS**: Mac OS X 10.12.3
**Thrown From**: [seti-ui](https://github.com/jesseweed/seti-ui) package 1.8.0


### Stack Trace

Uncaught TypeError: Cannot read property '0' of undefined

```
At /Users/xo/.atom/packages/seti-ui/lib/headers.coffee:8

TypeError: Cannot read property '0' of undefined
    at updateTreeViewHeaderPosition (/packages/seti-ui/lib/headers.coffee:8:23)
    at /packages/seti-ui/lib/headers.coffee:29:5
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:1)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:1)
    at Project.module.exports.Project.removePath (/Applications/Atom.app/Contents/Resources/app/src/project.js:322:1)
    at TreeView.module.exports.TreeView.removeProjectFolder (/Applications/Atom.app/Contents/Resources/app/node_modules/tree-view/lib/tree-view.js:1252:1)
    at HTMLDivElement.atom.commands.add.tree-view:remove-project-folder (/Applications/Atom.app/Contents/Resources/app/node_modules/tree-view/lib/tree-view.js:385:1)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:265:1)
    at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:3:1)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:166:1)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchContextMenuCommand (/Applications/Atom.app/Contents/Resources/app/src/atom-environment.js:1306:1)
    at EventEmitter.outerCallback (/Applications/Atom.app/Contents/Resources/app/src/application-delegate.js:347:1)
    at emitThree (events.js:116:13)
    at EventEmitter.emit (events.js:194:7)
```

### Commands

```
     -4:45.3.0 core:move-down (input.hidden-input)
     -4:45 editor:move-to-first-character-of-line (input.hidden-input)
     -4:44.5.0 core:move-right (input.hidden-input)
     -4:44.2.0 core:backspace (input.hidden-input)
     -4:43.7.0 editor:move-to-end-of-screen-line (input.hidden-input)
     -4:43.2.0 core:move-to-bottom (input.hidden-input)
     -4:42.3.0 core:select-to-top (input.hidden-input)
     -4:42 core:cut (input.hidden-input)
     -4:39.9.0 core:paste (input.hidden-input)
     -4:20.5.0 editor:consolidate-selections (input.hidden-input)
  2x -3:46 core:move-right (input.hidden-input)
     -1:49.2.0 application:add-project-folder (ol.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -1:35.1.0 tree-view:show (atom-workspace.workspace.scrollbars-visible-when-scrolling.seti-compact.seti-icons.seti-roboto.seti-animate.theme-seti-syntax.theme-seti-ui)
     -0:21.7.0 core:backspace (input.hidden-input)
     -0:13.7.0 core:save (input.hidden-input)
     -0:03.0 tree-view:remove-project-folder (span.name.icon.icon-repo)
```

### Non-Core Packages

```
autohide-tree-view 0.25.1 
busy-signal 1.4.1 
duplicate-removal 0.1.3 
editorconfig 2.2.2 
file-icons 2.1.7 
git-status 0.3.4 
intentions 1.1.2 
language-docker 1.1.8 
language-mako 0.6.0 
language-nginx 0.8.0 
language-pug 0.0.21 
language-vue 0.22.1 
linter 2.1.4 
linter-pycodestyle 2.0.2 
linter-stylelint 3.10.0 
linter-swagger 0.4.1 
linter-ui-default 1.6.0 
linter-xo 0.23.0 
multi-cursor 2.1.5 
seti-syntax 1.1.3 
seti-ui 1.8.0 
sort-lines 0.14.0 
tree-view-autoresize 2.1.0 
tree-view-git-status 1.4.0 
```

**Comment by jesseweed** [comment]
This should be fixed in the 1.9 release. Please let me know if you still experience the issue after updating. Thanks!
